pipelines/_Forced.yaml

```diff
@@ -8,4 +8,3 @@ subsets:
       - forcedPhotCcd
       - forcedPhotCoadd
     description: A set of tasks to run when doing forced measurements
```

Comment:
Does this pass github checks? I though yaml lint required a new line at the end (but maybe I have that backwards)

Reply:
lint DOES require a newline at the end. You had 2 newlines before: https://github.com/lsst/pipe_tasks/blob/master/pipelines/_Forced.yaml